src/main/java/uk/gov/hmcts/reform/jobscheduler/config/QuartzConfiguration.java

```diff
@@ -1,21 +1,35 @@
 package uk.gov.hmcts.reform.jobscheduler.config;
 
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import javax.inject.Singleton;
 import org.quartz.Scheduler;
 import org.quartz.SchedulerException;
 import org.quartz.impl.StdSchedulerFactory;
+import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import javax.inject.Singleton;
 
 @Configuration
+@ConfigurationProperties
 public class QuartzConfiguration {
 
+    private final Map<String, String> quartzProperties = new HashMap<>();
```

Comment:
is this automagically set by Spring based on field name?

Reply:
It won't work without the getter, even if the field is public. `@Value` will not work here, either, due to the fact that it's a map.